src/components/index/createRequestsTableCells.tsx

```diff
@@ -35,26 +35,7 @@ function createRequestsTableCells(requests: oracle.types.state.RequestIndexes) {
 
   if (requests.length) {
     requests.forEach((req) => {
-      let identifier = "";
-      try {
-        identifier = parseIdentifier(req.identifier);
-        if (identifier === "YES_OR_NO_QUERY") {
-          try {
-            identifier = formatYesNoQueryString(
-              ethers.utils.toUtf8String(req.ancillaryData)
-            );
-          } catch (err: any) {
-            console.error(
-              "error with formatYesNoQueryString call",
-              err.message
-            );
-          }
-        }
-      } catch (err: any) {
-        identifier = req.identifier;
-        console.log("Error in parsing identifier", err.message);
-      }
-
+      let identifier = formatRequestTitle(req);
       const timestamp = DateTime.fromSeconds(Number(req.timestamp)).toFormat(
```

Comment:
Nit: can probably make this a constant, as you are not reassigning the value in the file.

Reply:
good call